scripts/globals/spells/cure_vi.lua

```diff
@@ -62,6 +62,9 @@ function onSpellCast(caster,target,spell)
             else
                 solaceStoneskin = math.floor(final * 0.25)
             end
+            if caster:getMainJob() == dsp.job.WHM and caster:getMainLvl() >= 75 then
```

Comment:
Remove main job and main level check. They are not needed. getMerit handles this.